cfpb/cfgov-refresh
# Removes unused spanish language templates

## Removals

- Old spanish langage templates. These now live in the CMS

## Testing

1. Navigate to localhost:8000/es
2. Navigate to whatever spanish subpages you like (http://localhost:8000/es/comprar-casa/, http://localhost:8000/es/quienes-somos/, etc)

These pages should exist and be formatted correctly.
